bootstrap.sh

```diff
@@ -207,19 +207,6 @@ build_lib() {
       do_copy "sqlite3"
       do_build "sqlite3"
       ;;
-    vjson)
-      do_copy "vjson"
-      patch_external "vjson"       "missing_include.patch"
```

Comment:
I think the patch is now missing

Reply:
Yes, but it doesn't seem to be needed. Should I patch the file in place and drop the patch file?

Reply:
I think that would work